apps/example-app/src/app/home/components/default-phone-layout/default-phone-layout.component.html

```diff
@@ -1,5 +1,4 @@
-<section class="phone">
```

Comment:
Removed the section because there's no need for an additional wrapper.  Everything can be condensed to a single class in this case.